### PR TITLE
Db/songs: Creates routes for songs

### PR DESCRIPTION
**Pull request by zkodjo**
- Creates routes to get songs by artist
- Create route to add songs one at a time
- Create route to get one song

### CODE REVIEW

File: server/controllers/songs.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    next();
+  })
+    .catch((err) => {
+      logger.error(err);
+      res.status(500).send(`There was an error fetching song ${songId}`);
+      next(err);
+    });
+};
+
+exports.addOneSong = (req, res, next) => {
```
Comment (kalebal) [inline server/controllers/songs.js:57]: In order to embed a media player, we need the song's URL -- is that something we can add to the schema?

**Review by kalebal — CHANGES_REQUESTED** [review]

File: server/controllers/songs.js (hunk 1 above)
Comment (zkodjo) [reply server/controllers/songs.js:57]: Added some commits for this, take a look

Comment (kalebal) [reply server/controllers/songs.js:57]: Awesome, thank you!

**Review by kalebal — APPROVED** [review]
Looks great, thanks!

**Pull request merged**